src/OpenSage.Game/Data/FileSystem.cs

```diff
@@ -53,7 +53,7 @@ public FileSystem(string rootDirectory, FileSystem nextFileSystem = null)
 
         private void AddBigArchive(string path)
         {
-            var archive = new BigArchive(path);
+            var archive = new BigArchive(path, BigArchiveMode.Read);
```

Comment:
Actually I'd prefer the default is changed to `.Read`, not `.Update`. `ZipArchive` defaults to `ZipArchiveMode.Read` for example.

Reply:
True, not sure why i picked Update as the default mode, i was actually looking at the ZipArchive